examples/resources/cloudflare_pages_project/resource.tf

```diff
@@ -3,6 +3,10 @@ resource "cloudflare_pages_project" "basic_project" {
   account_id        = "f037e56e89293a057740de681ac9abbe"
   name              = "this-is-my-project-01"
   production_branch = "main"
+   deployment_configs {
+      preview {}
```

Comment:
this is bad form - we shouldn't ever use empty blocks as denotions for something. if the block is there, it should be populated with attributes.

Reply:
I wasn't sure the best way to handle the schema for this, as everything nest under the `deployment_configs` is computed.

Reply:
in that case, `deployment_configs` shouldn't be here at all since the only properties are computed.